src/Common/src/Interop/Windows/winmm/Interop.MMCKINFO.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class WinMM
+    {
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
```

Comment:
Ditto.  For that matter the whole StructLayout isn't necessary, as Sequential is the default.

Reply:
This is a class, so the default struct layout is Auto. But making this a struct would likely be an improvement.

Reply:
Ah, you're right, I didn't see that this was a class... I'm not used to seeing that in interop :)